packages/webviz-core/package-lock.json

```diff
@@ -19,11 +19,6 @@
         }
       }
     },
-    "@cruise-automation/hooks": {
```

Comment:
Leave this commit out? The `webviz-core` directory is updated by mirroring the changes from our internal repo.